packages/sources/twosigma/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@chainlink/twosigma-adapter",
+  "version": "1.0.0",
+  "description": "Chainlink twosigma adapter.",
+  "keywords": [
+    "Chainlink",
+    "LINK",
+    "blockchain",
+    "oracle",
+    "twosigma"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "repository": {
+    "url": "https://github.com/smartcontractkit/external-adapters-js",
+    "type": "git"
+  },
+  "license": "MIT",
+  "scripts": {
+    "clean": "rm -rf dist && rm -f tsconfig.tsbuildinfo",
+    "prepack": "yarn build",
+    "build": "tsc -b",
+    "server": "node -e 'require(\"./index.js\").server()'",
+    "server:dist": "node -e 'require(\"./dist/index.js\").server()'",
+    "start": "yarn server:dist"
+  },
+  "dependencies": {
+    "@chainlink/external-adapter-framework": "0.14.5",
```

Comment:
This version is a few iterations behind, we should bump it to the latest one

Reply:
Hi @alejoberardino, I attempted to bump to the latest version but I'm running into a framework issue that I don't think I can work around. Let me know if I'm missing something: https://github.com/smartcontractkit/ea-framework-js/issues/51.